packages/context/package.json

```diff
@@ -68,7 +68,7 @@
     },
     "dependencies": {
         "@microsoft/signalr": "^7.0.12",
-        "axios": "^1.6.7",
+        "axios": "1.7.3",
```

Comment:
I see this https://security.snyk.io/package/npm/axios

Why just upgrade to 1.7.3?

Reply:
I did the latest version then it causes the tests to fail with Invalid URL. 
There is a breaking change in 1.7.4+ versions. https://github.com/axios/axios/issues/6580